magic/core/src/main/java/link/magic/android/core/relayer/WebViewWrapper.kt

```diff
@@ -38,6 +37,7 @@ class WebViewWrapper internal constructor(context: Context, private val urlBuild
 
     internal val mMutableContext = MutableContextWrapper(context)
     private val webView: WebView = WebView(mMutableContext)
+    private var webViewDialog: WebViewDialog? = null
```

Comment:
I am not a fan of how I'm initializing the `WebViewDialog` here, but alas my Android achilles reveals it self. 😞 

As always, open to any suggestion you might have here.